### ISSUE TITLE
`yarn showcase-data:gen` failing in daily cron script

### ISSUE THREAD

**Issue by TylerAPfledderer** [chakra-ui/chakra-ui-docs#711]
### Description

Script `yarn showcase-data:gen` fails with the following error stack trace inside the `daily cron script` workflow:
```
Run yarn showcase-data:gen
yarn run v1.22.18
$ ts-node scripts/get-showcase-data.ts
(node:2131) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '_launcher' of undefined
    at launch (/home/runner/work/chakra-ui-docs/chakra-ui-docs/node_modules/puppeteer/src/node/Puppeteer.ts:155:17)
    at /home/runner/work/chakra-ui-docs/chakra-ui-docs/scripts/get-showcase-data.ts:36:31
    at step (/home/runner/work/chakra-ui-docs/chakra-ui-docs/node_modules/tslib/tslib.js:144:27)
    at Object.next (/home/runner/work/chakra-ui-docs/chakra-ui-docs/node_modules/tslib/tslib.js:125:57)
    at /home/runner/work/chakra-ui-docs/chakra-ui-docs/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/home/runner/work/chakra-ui-docs/chakra-ui-docs/node_modules/tslib/tslib.js:114:16)
    at main (/home/runner/work/chakra-ui-docs/chakra-ui-docs/scripts/get-showcase-data.ts:19:20)
    at Object.<anonymous> (/home/runner/work/chakra-ui-docs/chakra-ui-docs/scripts/get-showcase-data.ts:355:3)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2131) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2131) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 3.43s.
```

This results in the [Showcase Page](https://chakra-ui.com/showcase) not rendering up-to-date lists from the [awesome-chakra-ui repo](https://github.com/chakra-ui/awesome-chakra-ui)

**Comment by noobinthisgame** [comment]
Probably mb! @TylerAPfledderer a fix might be to downgrade pupeteer again.

**Comment by TylerAPfledderer** [comment]
@noobinthisgame I did incremental rollbacks all the way to `v8` and the error still occurred.

I actually found [this StackOverflow answer](https://stackoverflow.com/a/70257338) that suggested not importing `launch` as a named import. I tried that, and the script appeared to run successfully!

So from this:
```ts
import { launch } from 'puppeteer'
```
to this:
```ts
import Puppeteer from 'puppeteer'
```

And then updated [line 36](https://github.com/chakra-ui/chakra-ui-docs/blob/5bf8779072ce8e76ab6a0007fb38a28b7d8771d1/scripts/get-showcase-data.ts#L36) to the following:
```ts
const browser = await Puppeteer.launch({
  defaultViewport: {
    width: DEFAULT_VIEWPORT_WIDTH,
    height: DEFAULT_VIEWPORT_HEIGHT,
  },
})
```

Though I will now see the warning: `Caution: "Puppeteer" also has a named export "launch". Check if you meant to write "import {launch} from 'puppeteer'" instead.` 🤷🏼‍♂️ 

The updated json and `public` folder are showing the most current additions for the page.
